src/libnvme.map

```diff
@@ -1,7 +1,5 @@
 LIBNVME_1_0 {
 	global:
-		__nvme_get_log_page;
-		__nvme_msg;
```

Comment:
Can't remove this either at this point. The nvme-cli uses the nvme_msg() in fabrics.c. I think we should remove the use of nvme_msg in fabrics.c. nvme-cli should use it's own error logging macro.

Reply:
I removed the nvme_msg() calls in nvme-cli. So we can remove the __nvme_msg from the export map.